command/monitor.go

```diff
@@ -285,21 +285,23 @@ func (m *monitor) monitor(evalID string) int {
 		break
 	}
 
-	// Monitor the deployment
+	// Monitor the deployment if it exists
 	dID := m.state.deployment
-	m.ui.Info(fmt.Sprintf("%s: Monitoring deployment %q", formatTime(time.Now()), limit(dID, m.length)))
+	if len(dID) > 0 {
```

Comment:
Typically for a string field we'd just do something like:

```suggestion
	if dID != "" {
```

This disambiguates it from being a check on the length of a collection.